django_session_jwt/middleware/__init__.py

```diff
@@ -1 +1 @@
-from .session import SessionMiddleware
+from .session import SessionMiddleware, convert_cookie
```

Comment:
This would need `verify_jwt` for the import in `test.py` to work

Reply:
I fixed the import.